internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -27,47 +27,27 @@ func (opts *Opts) askConfirmConfigQuestion() error {
 		return nil
 	}
 
-	diskSize := 0.5
-	if opts.newCluster().DiskSizeGB != nil {
-		diskSize = *opts.newCluster().DiskSizeGB
-	}
-
-	const noMsg = "No"
-	const yesMsg = "Yes"
-	loadSampleData := yesMsg
-	if opts.SkipSampleData {
-		loadSampleData = noMsg
-	}
-
-	runMongoShell := noMsg
-	if opts.runMongoShell {
-		runMongoShell = yesMsg
+	loadSampleData := ""
+	if !opts.SkipSampleData {
+		loadSampleData = `
+Load sample data:			Yes`
 	}
 
 	fmt.Printf(`
-[Summary of changes]
+[Confirm cluster settings]
 Cluster Name:				%s
-Cluster Tier:				%s
```

Comment:
[not blocking] quickstart let's you change the tier and disk size, not showing it may be trouble some for users changing it

Reply:
@boooczek @blva  I didn't know quickstart allows users to change tier and disk size before. Is that possible to show them conditionally? Like the load sample data one; if users changed the tier (not M0), then show it in the confirmation, if not, then hide it?

Reply:
Yes, we can do this for both tier and disk size, will add.

Reply:
@gssbzn I only see the option to pick tier, when tier is picked, the disk size changes, is it? can I simply check the tier in that case?

Reply:
disk size is pick based on tier https://github.com/mongodb/mongodb-atlas-cli/blob/32c7d76befea47eddd1825125ee915462c4f9b3c/internal/cli/atlas/quickstart/cluster_setup.go#L97-L102

Reply:
cool, makes sense, I'll display the disk size when tier is not the default one if that's OK